example.env

```diff
@@ -0,0 +1,23 @@
+# Make a copy of this file with the name .env and populate the file with your environment variables
+
+# Port number to run the server on
+PORT=3030
+
+# Twilio account sid
+ACCOUNT_SID=ACxxxxxxxxxxxxxx
+
+# Twilio API key
+API_KEY_SID=SKxxxxxxxxxxxxxx
+
+# Twilio API secret
+API_KEY_SECRET=xxxxxxxxxxxxxx
+
+# Twilio TwiML App sid where the Voice Request URL is set to
+# https://yourdomain/twilio-voice-dialer/twiml
+APP_SID=APxxxxxxxxxxxxxx
+
+# Caller ID
+CALLER_ID=+11234567890
+
+# Callback base URL
+CALLBACK_BASE_URL=https://foo.ngrok.dev
```

Comment:
Is the user expected to use ngrok? If so should we add usage to README?

Reply:
They are not expected to use ngrok. This is just an example. The customer can use whatever they want to expose their APIs.